packages/vue-components/src/Tabset.vue

```diff
@@ -24,10 +24,12 @@
           v-else
           :key="index"
           class="nav-item"
-          :header="t.headerRendered"
           :class="{active:t.active}"
           :disabled="t.disabled"
         >
+          <template #header>
+            <span v-html="t.headerRendered"></span>
+          </template>
```

Comment:
Instead of making this change (since we rely on the conversions from slots to templates, and do not use header templates directly in our `.vue` files), a suggestion I have is to treat the `header` as a prop instead?

In `Dropdown.vue`, we can first include the `header` as a prop:

```
props: {
  ...
  header: {
    type: String,
    default: '',
  },
  ...
```

And then we can replace the `<slot name="header"></slot>` under the `v-if`, with `{{ header }}`. We could also rename `header` to something like `tabgroup-header` to differentiate it as well.

This might be a less hacky solution, but if we can investigate the issue deeper and find a better solution, that would be best 👍